src/views/Player.vue

```diff
@@ -55,9 +53,12 @@ export default {
       ivideo_id: null,
       source: 'unknown',
       isFullscreen: true,
-      supported_browsers: ['Chrome', 'Chrome Mobile', 'Firefox', 'Firefox Mobile', 'Microsoft Edge'],
+      supported_browsers: ['Chrome', 'Chrome Mobile'],
       isBrowserSupported: true,
-      failsafe: ''
+      browserErrorHandlingValue: {
+        'failsafe_type': 'g-form',
+        'failsafe_url': ''
+      }
```

Comment:
hardcoding g-form for now - once we decide to change, we'll make the corresponding change in Plio Maker as well and fetch from there.

Reply:
nice 👍